src/mappings/Coordinator.ts

```diff
@@ -141,6 +142,30 @@ export function addYields(pool: Pool, block: ethereum.Block): Pool {
   pool.juniorYield30Days = yields30.junior
   pool.seniorYield30Days = yields30.senior
 
+  // The inception is defined by minimum of the origination of the first loan and 30 days since pool creation
```

Comment:
The 30 days since pool creation are not considered below, right? 

Remove this comment?

Reply:
As discussed, this is actually checked above: https://github.com/centrifuge/tinlake-subgraph/blob/96a4da2df6a02e62c68d63c1c927a4b5f61c4ec5/src/mappings/Coordinator.ts#L130